include/odp/api/spec/pool.h

```diff
@@ -271,6 +271,9 @@ typedef struct odp_pool_capability_t {
 
 	/** Vector pool capabilities */
 	struct {
+		/** odp_pool_param_t::vector::max_size should be power of two. */
+		odp_bool_t size_is_pow2;
```

Comment:
I do not see why this is needed. I would think the implementation could simply internally round up the size to the next power of two.